packages/gatsby/src/utils/page-data.js

```diff
@@ -35,7 +36,50 @@ const write = async ({ publicDir }, page, result) => {
   await fs.outputFile(filePath, bodyStr)
 }
 
+const getChangedPageDataKeys = (
+  store,
```

Comment:
NIT: 
```suggestion
  state,
```
I was about to mention that `store.webpackCompilationHash` (and other `store.foo`) won't work, but then I realized it's actually `state` being passed here and not store ;)

Reply:
🤦‍♂ , thanks that's much clearer now